src/app/components/client/FixNavigation.tsx

```diff
@@ -0,0 +1,245 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import styles from "./FixNavigation.module.scss";
+import stepDoneIcon from "./assets/step-counter-done.svg";
+import stepDataBrokerProfilesIcon from "./assets/step-counter-data-broker-profiles.svg";
+import stepHighRiskDataBreachesIcon from "./assets/step-counter-high-risk.svg";
+import stepLeakedPasswordsIcon from "./assets/step-counter-leaked-passwords.svg";
+import stepSecurityRecommendationsIcon from "./assets/step-counter-security-recommendations.svg";
+import { useState } from "react";
+// import { Session } from "next-auth";
+import { useL10n } from "../../hooks/l10n";
+
+type StepId =
+  | "dataBrokerProfiles"
+  | "highRiskDataBreaches"
+  | "leakedPasswords"
+  | "securityRecommendations";
+
+// export type Props = {
+//   user: Session["user"];
+// };
+
+// type NavigationItemsContent = {
+//   key: string,
+//   name: string,
+//   href: string,
+//   status: number
+// }
+
+// type NavigationItems = Array<NavigationItemsContent>
+
+export const FixNavigation = (props: Props) => {
```

Comment:
```
38:31  Warning: 'props' is defined but never used. Allowed unused args must match /^_/u.  @typescript-eslint/no-unused-vars
```

Also unclear on what "Props" is here since the definition above on L25 is commented out (yet ESLint didn't seemingly flag that as an issue?)

Reply:
I just noticed the squiggle line. Same thought - it's weird lint didn't catch this? Added a props definition. Revised in 35e256b2940aa5e7350173c609c32d5f1f14cf3d